gulpfile.js/tasks/build.js

```diff
@@ -3,8 +3,7 @@ const { src, dest, parallel } = require('gulp');
 const sass = require('gulp-sass');
 const rename = require('gulp-rename');
 const postcss = require('gulp-postcss');
-const cssnano = require('cssnano');
-
+const postcssConfig = require('../../postcss.config');
```

Comment:
For using plugins from our centralized postcss config, shared with the webpack setup.

Reply:
The PostCSS plugin does this automatically. You shouldn't need to pass along a specific config.